### PR TITLE
Update mobilenetwork-create-new-site with new ASE install procedure for BaseVM only

### PR DESCRIPTION
**Pull request by o-wb**
# PR Checklist

Check these items before submitting a PR... 

[Contribution Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/README.md)

[Best Practice Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md)

- [x] - Please check this box once you've submitted the PR if you've read through the Contribution Guide and best practices checklist.

## Changelog

*
*
*

### CODE REVIEW

File: quickstarts/microsoft.mobilenetwork/mobilenetwork-create-new-site-basevm/azuredeploy.parameters.json [hunk 1]
```diff
@@ -15,17 +15,11 @@
             "value": "GEN-UNIQUE"
         },
         "platformType": {
```
Comment (andrew-lee-metaswitch) [inline quickstarts/microsoft.mobilenetwork/mobilenetwork-create-new-site-basevm/azuredeploy.parameters.json:17]: remove this

File: quickstarts/microsoft.mobilenetwork/mobilenetwork-create-new-site-basevm/main.bicep [hunk 2]
```diff
@@ -12,20 +12,14 @@ param siteName string = 'myExampleSite'
 
 @description('The platform type where packet core is deployed.')
 @allowed([
-  'AKS-HCI'
   'BaseVM'
 ])
-param platformType string = 'AKS-HCI'
+param platformType string = 'BaseVM'
```
Comment (andrew-lee-metaswitch) [inline quickstarts/microsoft.mobilenetwork/mobilenetwork-create-new-site-basevm/main.bicep:17]: remove this parameter

**Comment by bmoore-msft** [comment]
@o-wb can you check the merge conflicts?  Also, please remove azuredeploy.json from the PR.

**Comment by o-wb** [comment]
Abandoning this PR as agreed with Docs: we are deleting the BaseVM quickstarts entirely and only having them available internally.

**Pull request closed without merging**